components/applications-service/pkg/storage/postgres/service_groups.go

```diff
@@ -36,11 +36,15 @@ SELECT sg.id
       SELECT array_agg( DISTINCT CONCAT (s.origin, '/', s.name, '/', s.version, '/', s.release) )
       FROM service AS s
       WHERE s.group_id = sg.id
-    ) AS releases
+		) AS releases
```

Comment:
whitespace of despair :( I guess the rest of this is using spaces and your editor did tabs 🤷‍♀

Reply:
Lets just fix this indentation, please.

Reply:
My editor has betrayed me, I told it to do spaces!